spec/support/utils.rb

```diff
@@ -54,11 +54,15 @@ def self.delete_set(client, namespace, set_name)
 
   def self.client
     @client ||= begin
+      host = Aerospike::Host.new(
```

Comment:
Same as for the examples, I really don't think we should add new `AEROSPIKE_HOST` and `AEROSPIKE_PORT` env variables. Why can't you use the already supported `AEROSPIKE_HOSTS` env variable instead?